dependencies/src/main/resources/linkage-checker-exclusion-default.xml

```diff
@@ -0,0 +1,50 @@
+<LinkageCheckerFilter>
+  <!-- reactor-core's Traces catches Throwable to detect classes available in
+    Java 9+.
+    https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/816 -->
+  <LinkageError>
+    <Source>
+      <Package name="reactor.core.publisher.Traces" />
+    </Source>
+  </LinkageError>
+
+  <!-- GraalVM-related libraries depend on Java Compiler Interface (JVMCI) that
+    only exists in special JDK.
+    https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/929 -->
```

Comment:
Expand comment to explain why this is not a problem.

Reply:
Added explanation. It's not "why this is not a problem" but "why Linkage Checker reports false positives".

Reply:
The benefit of using elements and being flexible in what's allowed is that it enables people to fill needs we didn't plan for. It's the eXtensible part of XML.  We don't show these today. Tomorrow, who knows? It didn't occur to me when you designed this format that we might want documentation like this, bit now I can see the need.

Reply:
Also, I'm not sure this is correct. The reason is not that "As Linkage Checker does not use GraalVM, it ignores the errors to avoid false positives." Even if we did use GraalVM, I suspect this would happen. What we need here is an answer to the question of why do we consider this a false positive, and that answer is that the code is only invoked when running in a GraalVM that does have those classes.  In a non-Graal VM it's unreachable.

Reply:
Created an issue: https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1288 (not in this PR)


> Even if we did use GraalVM, I suspect this would happen.

If we use Java runtime that provides JVMCI Java Compiler Interface (JVMCI), this missing classes won't happen.

>  the code is only invoked when running in a GraalVM

Updated so.